SmartDeviceLink/SDLPreloadChoicesOperation.m

```diff
@@ -118,9 +118,18 @@ - (void)sdl_preloadCells {
 
     NSMutableArray<SDLCreateInteractionChoiceSet *> *choiceRPCs = [NSMutableArray arrayWithCapacity:self.cellsToUpload.count];
     for (SDLChoiceCell *cell in self.cellsToUpload) {
-        [choiceRPCs addObject:[self sdl_choiceFromCell:cell]];
+        SDLCreateInteractionChoiceSet *csCell =  [self sdl_choiceFromCell:cell];
+        if(csCell != nil) {
+            [choiceRPCs addObject:csCell];
```

Comment:
This should log an error as well

Reply:
what is the error?

Reply:
The developer is saying "show these cells," and this cell we're deciding not to show. We don't want to leave them to figure it out on their own.

Reply:
That error is being logged on line 127, line 121-123 add the item if the item is not nil

Reply:
Right, but that error is only logged if _none_ of the items are added. If the developer says, "show these 10 items," and one of them is missing a menu name, then a menu will only show up with 9 items. That's a problem if we don't log what went wrong and why only 9 items showed up instead of the 10 they asked for.

Reply:
added the error